fdbserver/Knobs.cpp

```diff
@@ -83,7 +83,9 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs) {
 	// Data distribution queue
 	init( HEALTH_POLL_TIME,                                      1.0 );
 	init( BEST_TEAM_STUCK_DELAY,                                 1.0 );
-	init( BG_DD_POLLING_INTERVAL,                               10.0 );
+	init(BG_REBALANCE_POLLING_INTERVAL, 10.0);
+	init(BG_REBALANCE_SWITCH_CHECK_INTERVAL, 5.0);
```

Comment:
Code style. 
To keep the code consistent and easier to read, the `Knobs.cpp` and `Knobs.h` files should not blindly use clang-format.

Reply:
I fixed this after the rebase. clang format is now turned off on `Knobs.cpp`